Cargo.toml

```diff
@@ -8,7 +8,7 @@ authors = [
 ]
 build = "build.rs"
 exclude = ["Gir*.toml", "tests/**/*", "*.md", "generator.py"]
-edition = "2021"
+edition = "2018"
```

Comment:
Why reverting this?

Reply:
For rust analyzer, I wasn't meaning to push it

Reply:
Why? For me it works with 2021 :)

Reply:
Question remains. :)

Reply:
Still waiting.